runtime/binding-kafka/src/main/java/io/aklivity/zilla/runtime/binding/kafka/internal/stream/KafkaMergedFactory.java

```diff
@@ -1952,6 +1971,7 @@ private void onTopicConfigChanged(
             long traceId,
             ArrayFW<KafkaConfigFW> configs)
         {
+            configs.forEach(c -> this.configs.put(c.name().asString(), c.value().asString()));
```

Comment:
Do we need to store all of them?

Lets change `configs` to be keyed by `StringFW` and just keep `CONFIG_NAME_CLEANUP_POLICY` config value in the map, then we'll also avoid multiple unnecessary calls to `.asString()`.